src/server/Server/Data/ServerUser.cs

```diff
@@ -0,0 +1,8 @@
+using Microsoft.AspNetCore.Identity;
+
+namespace Server.Data;
+
```

Comment:
Do we really need a seperate user for this? Any chance we can use the SystemUser in the future?

Reply:
Das is am server, auf azure, für plugin autoren, ganz eigene  welt